utils/components-testkit/src/main/scala/pl/touk/nussknacker/engine/util/test/TestScenarioRunner.scala

```diff
@@ -50,6 +50,8 @@ trait TestScenarioRunnerBuilder[R <: TestScenarioRunner, B <: TestScenarioRunner
 
   def withExtraComponents(components: List[ComponentDefinition]): B
 
+  def withGlobalVariables(globalVariables: Map[String, AnyRef]): B
```

Comment:
Let's be consistent: `withExtraGlobalVariables`. It will also be more clear, that its are not the only variables, we also add the default onece

Reply:
Fix docs as well - there is still old version there

Reply:
You check changes to fast.. :) I haven't event checked changes on the diff..